src/assets/images/add-button.svg

```diff
@@ -0,0 +1,5 @@
+<svg viewBox="0 0 24 24" width="24" height="24">
```

Comment:
Sab cheez ke liye SVG? We should see if it's possible with pure CSS. Will probably be possible! Not for this PR though.

SVGs are in general more computationally expensive

Reply:
I'll keep this in mind next time.